aip/general/0111.md

```diff
@@ -0,0 +1,90 @@
+---
+id: 111
+state: draft
+created: 2023-06-10
+placement:
+  category: api-concepts
+  order: 50
+---
+
+# Planes
+
+Resources and methods on an API can be divided into the *plane* that they reside
+or perform operations upon. For the context of APIs, the following planes are
+defined:
+
+- The management plane: a uniform, resource-oriented API that primarily
+  configures and allows retrieval of resources.
+- The data plane: a heteregenous API (ideally resource-oriented) that interface
+  with the resources that the the management plane configures.
+
+The term "plane" was originally used in networking architecture. Although system
+and network architecure often defines additional planes (e.g. control plane or
+power planes), as the AIPs are focused on the interface, they are not defined in
+this AIP.
+
+## Guidance
+
+### Management Plane
+
+Management resources and methods exist primarily to provision, configure, and
+audit the resources that the data plane interfaces with.
+
+As an example, the following are considered management resources for a cloud
+provider:
+
+- virtual machines
+- virtual private networks
+- disks
+- a blob store instance
+- a project or account
+
+The following behaviors are common (but not required) attributes of a
+management plane resource:
+
+- changes infrequently.
```

Comment:
This won't be true in the case of control loops, such as autoscaling, built on the standard resource APIs.

Reply:
We need consistent terminology for referring to (a) the resource attributes, sometimes known as metadata, but that's ambiguous since fields like name, creation time, and annotations are also referred to as metadata, and (b) user data.

Reply:
for now I can remove this section - we can always add something to that effect later, and I think the "major distinctions" section does a good job highlighting key differences.